error[E0425]: cannot find function `generate_header` in crate `taskchampion_lib` in release mode

```
chromy@astra ~/s/taskchampion (main) [101]> cargo --version
cargo 1.77.2 (e52e36006 2024-03-26)
chromy@astra ~/s/taskchampion (main)> rustc --version
rustc 1.77.2 (25ef9e3d8 2024-04-09)
chromy@astra ~/s/taskchampion (main)> cargo build -r
   Compiling xtask v0.5.0 (/home/chromy/src/taskchampion/xtask)
error[E0425]: cannot find function `generate_header` in crate `taskchampion_lib`
   --> xtask/src/main.rs:50:49
    |
50  |     write!(&mut file, "{}", ::taskchampion_lib::generate_header()).unwrap();
    |                                                 ^^^^^^^^^^^^^^^ not found in `taskchampion_lib`
    |
note: found an item that was configured out
   --> /home/chromy/src/taskchampion/lib/src/lib.rs:170:8
    |
170 | pub fn generate_header() -> String {
    |        ^^^^^^^^^^^^^^^

For more information about this error, try `rustc --explain E0425`.
error: could not compile `xtask` (bin "xtask") due to 1 previous error
```

--- a/lib/src/lib.rs
+++ b/lib/src/lib.rs
@@ -165,7 +165,6 @@ pub(crate) mod types {
     pub(crate) use crate::workingset::TCWorkingSet;
 }
 
-#[cfg(debug_assertions)]
 /// Generate the taskchapion.h header
 pub fn generate_header() -> String {
     ffizz_header::generate()

